internal/monitors/collectd/postgresql/postgresql.go

```diff
@@ -89,11 +96,11 @@ type Database struct {
 	// Specify the Kerberos service name used to authenticate with kerberos 5 or
 	// GSSAPI
 	KRBSrvName string `yaml:"krbSrvName"`
-	// Queries used to generate metrics.  If no queries are specified, the
-	// default set will be used [`custom_deadlocks`, `backends`, `transactions`,
-	// `queries`, `queries_by_table`, `query_plans`, `table_states`,
-	// `query_plans_by_table`, `table_states_by_table`, `disk_io`,
-	// `disk_io_by_table`, `disk_usage`]
+	// Queries used to generate metrics. These will override the default set.
+	// If no queries are specified, the default set will be used
+	// [`custom_deadlocks`, `backends`, `transactions`, `queries`, `queries_by_table`,
+	// `query_plans`, `table_states`, `query_plans_by_table`, `table_states_by_table`,
+	//  `disk_io`, `disk_io_by_table`, `disk_usage`]
```

Comment:
I think the extra space here is breaking the script that generates the md

Reply:
Yes, it is.

Reply:
Removed the space